tests/fixtures.py

```diff
@@ -5,16 +5,34 @@
 
 import pytest
 
-from zeusops_bot.models import ConfigFile, ModDetail
+from zeusops_bot.models import ConfigFile
 from zeusops_bot.reforger_config_gen import ReforgerConfigGenerator
 
 BASE_CONFIG: ConfigFile = {"game": {"scenarioId": "old-value", "mods": []}}
 
-MODLIST: list[ModDetail] = [
-    {"modId": "595F2BF2F44836FB", "name": "RHS - Status Quo", "version": "0.10.4075"},
-    {"modId": "5EB744C5F42E0800", "name": "ACE Chopping", "version": "1.2.0"},
-    {"modId": "60EAEA0389DB3CC2", "name": "ACE Trenches", "version": "1.2.0"},
-]
+MODLIST: str = """
+    {
+        "modId": "595F2BF2F44836FB",
+        "name": "RHS - Status Quo",
+        "version": "0.10.4075"
+    },
+    {
+        "modId": "5EB744C5F42E0800",
+        "name": "ACE Chopping",
+        "version": "1.2.0"
+    },
+    {
+        "modId": "60EAEA0389DB3CC2",
+        "name": "ACE Trenches",
+        "version": "1.2.0"
+    }
+"""
+
+FIRST_MOD = {
+    "modId": "595F2BF2F44836FB",
+    "name": "RHS - Status Quo",
+    "version": "0.10.4075",
+}
```

Comment:
Per other comments, suggest to revert that for the test of business logic, but say keep this as a new, standalone test of `string -> list[ModDetail]` convertion!

Reply:
Agreed, could test both formats

Reply:
Added both `MODLIST` formats side by side, the test now converts the JSON using `extract_mods()` and generates the config based on the result.